bitnami/haproxy/Chart.yaml

```
@@ -28,4 +28,4 @@ maintainers:
 name: haproxy
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/haproxy
-version: 1.0.4
+version: 1.0.5
```

As you are adding a new parameter, you need to bump the chart to a new minor version

```suggestion
version: 1.1.0
```